lib/screens/package_details_page.dart

```diff
@@ -0,0 +1,341 @@
+import 'package:dynamic_theme/dynamic_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:groovin_material_icons/groovin_material_icons.dart';
+import 'package:groovin_widgets/groovin_widgets.dart';
+import 'package:provider/provider.dart';
+import 'package:pub_client/pub_client.dart' hide Tab;
+import 'package:pub_dev_client/src/pub_colors.dart';
+import 'package:pub_dev_client/widgets/html_view.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'package:rounded_modal/rounded_modal.dart';
+
+class PackageDetailsPage extends StatefulWidget {
+  static const routeName = '/PackageDetailsPage';
+
+  @override
+  _PackageDetailsPageState createState() => _PackageDetailsPageState();
+}
+
+class _PackageDetailsPageState extends State<PackageDetailsPage> with SingleTickerProviderStateMixin {
+  TabController _tabController;
+  ScrollController _scrollViewController;
+  PubHtmlParsingClient _htmlParsingClient = PubHtmlParsingClient();
+
+  @override
+  void initState() {
+    _tabController = TabController(length: 7, vsync: this);
+    _scrollViewController = ScrollController();
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    _scrollViewController.dispose();
+    _tabController.dispose();
+    super.dispose();
+  }
+
+
+  @override
+  Widget build(BuildContext context) {
+    final PackageDetailsArguments args = ModalRoute.of(context).settings.arguments;
+
+    Color scoreColor;
+    int score = int.parse(args.packageScore);
+    if (score <= 50 || score == null) {
+      scoreColor = Provider.of<PubColors>(context).badPackageScore;
+    } else if (score >= 51 && score <= 69) {
+      scoreColor = Provider.of<PubColors>(context).goodPackageScore;
+    } else {
+      scoreColor = Provider.of<PubColors>(context).greatPackageScore;
+    }
+
+    List<Tab> _tabs = [
```

Comment:
We can't hardcode the tabs section since it can vary. To make it futureproof, let's base it off of the tabs that are provided in the Page. Something like, inside the `FutureBuilder`
```
    List<Tab> _tabs = [for(final tab in package.tabs)Tab(
        child: Text(
          'tab.title',
          style: TextStyle(
            color: DynamicTheme.of(context).brightness == Brightness.light ? Colors.black : Colors.white,
          ),
        ),
      ),];
   
```

Reply:
gotcha

Reply:
This doesn't account for making the score widget